video_creation/data/videos.json

```diff
@@ -1 +0,0 @@
-[]
```

Comment:
Why is this exactly? The code will crash if this is removed

Reply:
It doesn't crash, I tested, reason for not crashing is that there's previously a line that creates this file if it doesn't exist

Reply:
There were a lot of issues with this if you completely delete the file contents, can you just revert it? Thanks!

Reply:
subreddit.py line 30 is creating this file before using, I deleted this file and running script again created it and throw no error, this file will probably keep creating issues only with PR requests of new contributors. also some admin was sure that this file wasnt needed so that probably why it exists in .gitignore twice. I have removed duplicate there.
are you still sure you want it back?

  
 # recursively checks if the top submission in the list was already done.
    if not exists("./video_creation/data/videos.json"):
        with open("./video_creation/data/videos.json", "w+") as f:
            json.dump([], f)
    with open(
        "./video_creation/data/videos.json", "r", encoding="utf-8"
    ) as done_vids_raw:
        done_videos = json.load(done_vids_raw)